src/libs/utils.js

```diff
@@ -377,3 +377,12 @@ export const formatBytes = bytes => {
   ].find(([_p, d]) => bytes >= d)
   return `${(bytes / divisor).toPrecision(3)} ${prefix}iB`
 }
+
+//Truncates an integer to the thousands, i.e. 10363 -> 10k
+export const truncateInteger = integer => {
+  if (integer < 1000) {
```

Comment:
Could we increase this to 10,000? I think for smaller numbers, for example if we have 1999, than displaying 1k would be a big shift. Do 4 digits fit?

Reply:
4 digits do fit! Addressed in c8e7208f19a8c8f05b7a020612c63304ffe32c5f